css/index.css

```diff
@@ -1 +1,337 @@
+  font-size: 7.2rem;
+  text-align: center;
+  letter-spacing: 5px;
+  display: flex;
+  justify-content: center;
+  flex-flow: column;
+  align-items: center;
+  margin: 0 9.5% 0 0;
+}
+
+.cta .cta-text button {
+  border: .1rem double black;
+  font-size: 1.6rem;
+  background: white;
+  width: 57%;
+  height: 32px;
+  margin-top: 10px;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+ border-top: 2px solid black; 
+ border-bottom: 2px solid black;
+ margin: 75px 0 35px 0;
+ padding: 35px 0;
+ font-size: 1.6rem;
+ line-height: 18px;
+}
+
+.main-content .top-content {
+  display: flex;
+}
+
+.main-content .middle-img {
+  display: block;
+  margin: 4% auto;
+}
+
+.main-content .bottom-content {
+  display: flex;
+}
+
+.main-content .bottom-content p {
+  padding-right: 2.5%;
+}
+
+/* Contact Styles */
+
+.contact h4 {
+    font-size: 1.6rem;
+}
+
+.contact p {
+    font-size: 1.6rem;
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+  text-align: center;
+  margin: 40px 0 20px;
+  }
+
+footer p {
+font-size: 1.6rem;
+}
+
+/* 800px media query */
+
+@media (max-width: 800px) {
+  .container {
+    width: 90%;
+    margin: 0 auto;
+  }
+  
+  header {
+    display: flex;
+    flex-flow: column-reverse;
+    align-items: center;
+  }
+
+  header .logo {
+    width: 26%;
+    margin: 8.5% 0 0;
+  }
+  header nav {
+    justify-content: space-between;
+    margin-top: 5%;
+    width: 102%;
+    display: flex;
+  }
+  .cta {
+    display: flex;
+    margin-top: 72px;
+    flex-flow: column;
+  }
+  
+  .cta .cta-text {
+    font-size: 7.2rem;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 1%;
+  }
+  
+  .cta .cta-text button {
+    border: .1rem double black;
+    font-size: 1.6rem;
+    background: white;
+    width: 25%;
+    height: 32px;
+    margin-top: 7px;
+    cursor: pointer;
+  }
+  .cta img {
+    display: none;
+  }
+
+  .main-content {
+    border-top: 2px solid black;
+    border-bottom: 2px solid black;
+    margin: 67px 0 38px 0;
+    padding: 49px 0;
+    font-size: 1.6rem;
+    line-height: 18px;
+  }
+
+  .main-content .middle-img {
+    display: block;
+    width: 100%;
+    margin: 3.75%px auto;
+  }
+
+  @media (max-width: 500px) {
+    .container {
+      margin: 0 auto;
+    }
+    
+    header {
+      display: flex;
+      flex-flow: column-reverse;
+      align-items: center;
+    }
+  
+    header .logo {
+        width: 44%;
+        margin: 2%;
+    }
+    header nav {
+      border: 1px black;
+      display: flex;
+      flex-flow: column;
+      margin-top: 3%;
+      width: 100%;
+      justify-content: space-evenly;
+    }
+    header nav a {
+        font-size: 20px;
+        padding-left: 41%;
+        padding-top: 19px;
+        padding-bottom: 19px;
+        border: 1px solid black;
```

Comment:
You can have the border-bottom: 1px solid black, and have a header nav:first-child { border-top: 1px solid black } to accomplish the single border look without overlapping.